src/promptflow-evals/samples/EvaluatorUpload.ipynb

```diff
@@ -0,0 +1,494 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6018523f-3425-4bb3-9810-d31b8912991c",
+   "metadata": {},
+   "source": [
+    "# Upload of evaluators\n",
+    "In this notebook we are demonstrating the upload of the standard evaluators.\n",
+    "\n",
+    "### Import"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d3e21ceb-1e58-4ba7-884a-5e103aea7ecc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import json\n",
+    "import pandas as pd\n",
+    "import shutil\n",
+    "import uuid\n",
+    "import yaml\n",
+    "\n",
+    "from azure.ai.ml import MLClient\n",
+    "from azure.identity._credentials.default import DefaultAzureCredential\n",
```

Comment:
why not "from azure.identity import DefaultAzureCredential"?

Reply:
Fixed